### DISCUSSION TITLE
update python3.8 to 3.9 or higher

### DISCUSSION THREAD

**Discussion by berndhskb** [Guilouz/Klipper-Flsun-Speeder-Pad#156] [Q&A]
Hello everyone, I would like to update python3, version 3.8 is currently installed. For spoolman I need at least version 3.9. have already
sudo apt install python3.9
sudo apt remove python3.8
carried out.
ls /usr/bin | grep python
arm-linux-gnueabihf-python2-config
arm-linux-gnueabihf-python2.7-config
arm-linux-gnueabihf-python3-config
arm-linux-gnueabihf-python3.8-config
dh_python2
python
python config
python2
python2 config
python2.7
python2.7-config
python3
python3.8
python3.9
who can help me?
Thanks

**Comment by Hydra20vt** [comment]
Same problem here. Have you found a solution?

I was able to install pyton 3.9 and change it as the used version on speederpad but this operation makes the KlipperScreen app not working anymore

**Reply by berndhskb** [reply]
I managed it, but unfortunately I forgot it again

**Reply by Hydra20vt** [reply]
I did it yesterday following the updated guide with new dependencies installation
